executor/tests/manual/test_api_connection.py

```
@@ -0,0 +1,121 @@
+from executor.services.updater.github_version_checker import GithubVersionChecker
+
+
+def test_api_connection():
+    """测试 API 连接"""
+    checker = GithubVersionChecker()
+    binary_name = checker.get_binary_name()
+
+    # 构建 API URL (for GitHub API)
+    api_url = f"{checker.API_BASE}/repos/{checker._get_github_repo()}/releases/latest"
+
+    print(f"测试 API 连接:")
```

Ruff (0.15.6)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)